explorer/interpreter/type_checker.cpp

```diff
@@ -4796,6 +4796,39 @@ auto TypeChecker::DeclareClassDeclaration(Nonnull<ClassDeclaration*> class_decl,
     class_vtable[fun->name().inner_name()] = {fun, class_level};
   }
 
+  // Check destructor's virtual override, add to vtable if necessary.
+  if (const auto destructor = class_decl->destructor()) {
+    const auto* fun = (*destructor);
+    llvm::StringRef destructor_name = "destructor";
```

Comment:
+`const`

Reply:
```suggestion
    static constexpr llvm::StringLiteral DestructorName = "destructor";
```

I think this may be more what you want...